types/absinthe__socket-apollo-link/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for @absinthe/socket-apollo-link 0.2
+// Project: https://github.com/absinthe-graphql/absinthe-socket/tree/master/packages/absinthe-phoenix-socket-apollo-link#readme
+// Definitions by: Maarten van Vliet <https://github.com/DefinitelyTyped>
```

Comment:
Use your GH username instead of `Defin itelyTyped`...

Reply:
Fixed!